src/controllers/order/index.ts

```diff
@@ -20,6 +37,83 @@ export class order extends Controller {
     this.db = new Database()
   }
 
+  /**
+   * Returns the details of all order demands.
+   * @summary List all order demands
+   */
+  @Get('/')
+  public async getAll(): Promise<DemandResponse[]> {
+    const order = await this.db.getDemands(DemandSubtype.order)
+    const result = await Promise.all(
+      order.map(async (order: DemandResponse) => ({
+        ...order,
+        owner: await getMemberByAddress(order.owner).then(({ alias }) => alias),
+      }))
+    )
+    return result
+  }
+
+  /**
+   * @summary Get a order by ID
+   * @param orderId The order's identifier
+   */
+  @Response<NotFound>(404, 'Item not found')
+  @Get('{orderId}')
+  public async getById(@Path() orderId: UUID): Promise<DemandResponse> {
+    const [order] = await this.db.getDemand(orderId)
+    if (!order) throw new NotFound('order')
+
+    return {
+      ...order,
+      owner: await getMemberByAddress(order.owner),
+    }
+  }
+
+  /**
+   * @summary Get a order creation transaction by ID
+   * @param orderId The order's identifier
+   * @param creationId The order's creation ID
+   */
+  @Response<NotFound>(404, 'Item not found.')
+  @SuccessResponse('200')
+  @Get('{orderId}/creation/{creationId}')
```

Comment:
there's the get all creation transactions route as well `@Get('{orderId}/creation/'`. Depends if you want to do in this PR

Reply:
Yea, I did not want to do transactions, but since I started, will put that in along with a test.